server/models/Appointment.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+const appointmentSchema = new mongoose.Schema({
+  meeting_id: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  name: {
+    type: String,
+    required: true
+  },
+  email: {
+    type: String,
+    required: true
+  },
+  time: {
+    type: Date,
+    required: true
+  },
+  timezone: {
+    type: Date,
+    required: true
+  }
```

Comment:
What is the plan with this here? Wouldn't it be better to store the users timezone as a string from a list of supported time zones in our Date library and then we can just do the conversion on the time property's date?

Reply:
That's a good point, I just pushed a commit that converts the type to a string. Thanks for the feedback!